tracee-rules/main.go

```diff
@@ -59,7 +62,17 @@ func main() {
 				}()
 			}
 
-			sigs, err := getSignatures(c.String("rules-dir"), c.StringSlice("rules"))
+			var target string
+			switch strings.ToLower(c.String("target")) {
+			case "wasm":
```

Comment:
I think that with OPA v0.32.0 we have to import `wasm` package for side effects. In previous version they used Go build tags. Did you add it somewhere?

```go
import (
	_ "github.com/open-policy-agent/opa/features/wasm"
)
```

Reply:
Yeah good catch - I had changed this in the benchmarks locally but forgot to do so here. I added it in now.